### PR TITLE
[Feat]: Add Multilingual Support for Timesheet Component

### PR DESCRIPTION
**Pull request by Innocent-Akim**
## Description

Please include a summary of the changes and the related issue.

## Type of Change

-   [x] Bug fix
-   [x] New feature
-   [ ] Breaking change
-   [ ] Documentation update

## Checklist

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings

## Previous screenshots

Please add here videos or images of previous status

## Current screenshots

Please add here videos or images of previous status

## Release Notes

- **New Features**
	- Enhanced button styling in the FilterWithStatus component for better visual distinction.
	- Improved layout flexibility in the FrequencySelect component.
	- Localization support added for button titles in TimesheetAction and TimesheetCard components.
	- Conditional rendering in TimesheetView to improve user feedback during data loading.
	- New localization strings added for various languages, enhancing the user experience across the application.

- **Bug Fixes**
	- Corrected syntax for text color classes in the FilterWithStatus component.
	- Fixed greeting translations in the date helper function.

- **Chores**
	- Added localization entries for timesheet functionalities in multiple language files.

### CODE REVIEW

File: apps/web/locales/de.json [hunk 1]
```diff
@@ -549,6 +549,19 @@
 		"maintenance": {
 			"HEADING_TITLE": "We zijn in onderhoud ",
 			"HEADING_DESCRIPTION": "Momenteel zijn wij onze website aan het vernieuwen om u nog beter van dienst te kunnen zijn. Kom later nog eens terug."
```
[the remark this answers is not in the document]
Comment (Innocent-Akim) [reply apps/web/locales/de.json:551]: no change

[the remark this answers is not in the document]
Comment (Innocent-Akim) [reply apps/web/locales/de.json:551]: no change

File: apps/web/locales/it.json [hunk 2]
```diff
@@ -549,6 +549,27 @@
...
+				"GOOD_AFTERNOON": "Buon pomeriggio",
+				"GOOD_EVENING": "Buona sera"
+			},
+			"VIEWS": {
+				"LIST": "Visualizzazione elenco",
+				"CALENDAR": "Visualizzazione calendario"
+			},
+			"BILLABLE": {
+				"YES": "Sì",
+				"NO": "No"
```
[the remark this answers is not in the document]
Comment (Innocent-Akim) [reply apps/web/locales/it.json:571]: no change

File: apps/web/app/[locale]/timesheet/[memberId]/components/FilterWithStatus.tsx [hunk 3]
```diff
@@ -25,9 +25,9 @@ export function FilterWithStatus({
                 <Button
                     key={index}
                     className={clsxm(
-                        'group flex items-center justify-start h-[2.2rem] rounded-xl border w-full',
+                        'group flex items-center justify-start h-[2.2rem] rounded-xl w-full',
                         'dark:bg-dark--theme-light dark:border-gray-700 bg-transparent text[#71717A] w-[80px]',
-                        activeStatus === label && 'text-primary bg-white shadow-lg font-bold'
+                        activeStatus === label && 'text-primary bg-white shadow-lg font-bold border'
```
[the remark this answers is not in the document]
Comment (evereq) [reply apps/web/app/[locale]/timesheet/[memberId]/components/FilterWithStatus.tsx:30]: @Innocent-Akim ^^^^ ?????

**Comment by evereq** [comment]
@Innocent-Akim please go over ALL suggestions by Coderabbit and either fix code or close those. Make sure you set PR draft after you get review by Coderabbit if you see you need to work on fixing more code etc. CC: @Cedric921 please make sure you do that same too.

**Pull request merged**